SensorNode/sensor_nodes/onBoxSensorNode.py

```diff
@@ -86,6 +91,7 @@ async def onRTPCommandResponse(self, packet: codec.binaryPacket):
                 f' -vcodec copy -f mpegts tcp://{self.data_endpoint}:{endpoint_port}')
             proc_out = asyncio.subprocess.PIPE
             proc_err = asyncio.subprocess.PIPE
+            self._log.info(f'Starting ffmpeg with command: {cmd}')
```

Comment:
It would also be good to know how ffmpeg exited, and what the error output was.  See the server code.